### ISSUE TITLE
Missing translation in compose new email

### ISSUE THREAD

**Issue by thomascube** [thomascube/roundcube-trac-import#34]
**Reported by moc on 21 May 2007 10:20 UTC**
In the file ''/skins/default/templates/compose.html'' the two labels Cc and Bcc are not translated. Here is the patch:

```
<td class="add-button"><a href="#" onclick="return rcmail_toggle_display('compose-cc')">[name="cc" />](<roundcube:label)</a><br />
<a href="#" onclick="return rcmail_toggle_display('compose-bcc')">[name="bcc" />](<roundcube:label)</a><br /></td>
```

Migrated-From: http://trac.roundcube.net/ticket/1484383

**Comment by thomascube** [comment]
**Comment by robin on 19 Sep 2007 13:06 UTC**
That's not a patch, that's modified original code :)

Anyway, I made the change, but the most-right <td> elements are set at 40px, which isn't enough for the dutch translation at least. The elements' width should be set automatically, but in my Firefox the textarea overlaps the text. So this needs more work.


```
Index: skins/default/templates/compose.html
===================================================================
--- skins/default/templates/compose.html	(revision 803)
+++ skins/default/templates/compose.html	(working copy)
@@ -63,8 +63,8 @@
 
 <td class="title top"><label for="rcmcomposeto"><roundcube:label name="to" /></label></td>
 <td><roundcube:object name="composeHeaders" part="to" form="form" id="rcmcomposeto" cols="80" rows="2" tabindex="2" /></td>
-<td class="add-button"><a href="#" onclick="return rcmail_toggle_display('compose-cc')">[/>
-<a href="#" onclick="return rcmail_toggle_display('compose-bcc')">[Bcc](Cc]</a><br)</a><br /></td>
+<td class="add-button"><a href="#" onclick="return rcmail_toggle_display('compose-cc')">[name="cc" />](<roundcube:label)</a><br />
+<a href="#" onclick="return rcmail_toggle_display('compose-bcc')">[name="bcc" />](<roundcube:label)</a><br /></td>
 
 </tr><tr id="compose-cc">
 
Index: skins/default/mail.css
===================================================================
--- skins/default/mail.css	(revision 803)
+++ skins/default/mail.css	(working copy)
@@ -785,7 +785,7 @@
 
 #compose-headers td.add-button
 {
-  width: 40px !important;
+  width: 60px !important;
   text-align: right;
   vertical-align: bottom;
 }
```

**Comment by thomascube** [comment]
**Comment by thomasb on 26 Sep 2007 08:15 UTC**
Implemented a slightly different solution with new texts but this issue is considered to be fixed in [7984ece2].
